RetailCoder.VBE/Settings/HotkeySettings.cs

```diff
@@ -55,14 +55,14 @@ public HotkeySetting[] Settings
                 foreach (var setting in incoming.GroupBy(s => s.Name).Select(hotkey => hotkey.First()))
                 {
                     //Only allow one hotkey to be enabled with the same key combination.
-                    setting.IsEnabled &= !IsDuplicate(setting);
+                    setting.IsEnabled &= !IsDuplicate(setting) && setting.IsValid;
```

Comment:
you're starting to make the comments lie. I'd code the `IsValid` into the LINQ query.

Reply:
@Vogel612  `foreach (var setting in incoming.Where(s => s.IsValid).GroupBy(s => s.Name).Select(hotkey => hotkey.First()))` gets wordy though. no?